components/chatbot/controller.js

```diff
@@ -201,78 +173,18 @@ function Controller() {
     }
   };
 
-  async function getChatStatus(chatId) {
-    try {
-      const response = await axios.get(
-        `/api/chatbot/getChatStatus?chat_id=${chatId}`,
-        {
-          headers: {
-            Authorization: `Bearer ${accessToken || ""}`,
-          },
-        }
-      );
-      const chatStatus = response.data.chatStatus;
-      return chatStatus;
-    } catch (error) {
-      return "";
-    }
-  }
-
-  async function displayRelevantFile(chatId, inputText, aiResponse) {
-    let clientId = currentClient.uuid;
-    const body = {
-      chat_id: chatId,
-      client_id: clientId,
-      human_message: inputText,
-      ai_message: aiResponse,
-    };
-    try {
-      const response = await axios.post(
-        "/api/chatbot/getDisplayRelevantFile",
-        body,
-        {
-          headers: {
-            Authorization: `Bearer ${accessToken}`,
-            "Content-Type": "application/json",
-          },
-        }
-      );
-      if (response.status === 200) {
-        const releventFile = response.data;
-        console.log("Relevent file is : ", releventFile);
-        setFileObject(releventFile);
-        return releventFile;
-      }
-    } catch (error) {
-      return setFileObject({
-        file_id: -1,
-        file_name: "No relevant file found",
-      });
-    }
-  }
-
-  async function getRelevantFile(chatId, inputText, aiResponse) {
-    let clientId = currentClient.uuid;
-    console.log("getRelevantFile");
-    console.log("chatId", chatId);
-    console.log("inputText", inputText);
-    console.log("aiResponse", aiResponse);
+  async function getRelevantFile(chatId, inputText, aiResponse, isDisplay = true) {
     const body = {
       chat_id: chatId,
-      client_id: clientId,
+      client_id: currentClient.uuid,
       human_message: inputText,
       ai_message: aiResponse,
     };
     try {
-      const response = await axios.post("/api/chatbot/getRelevantFile", body, {
-        headers: {
-          Authorization: `Bearer ${accessToken}`,
-          "Content-Type": "application/json",
-        },
-      });
+      const request = isDisplay ? getDisplayRelevantFileAPI : getRelevantFileAPI;
```

Comment:
As previously mentioned, the decision of which REST API to use will be determined by the final parameter.